src/components/Proposals/ProposalDetails/StatusInfo.js

```diff
@@ -14,7 +14,7 @@ function StatusInfo({ vote }) {
 
   const infoDescription = {
     [VOTE_STATUS_DISPUTED]: (
-      <InfoBox>
+      <InfoBox background="#FFEAEA" borderColor="#FF7C7C" color="#FF7C7C">
```

Comment:
Can we use the `error` mode instead?

Reply:
Mmmhh, I dont think so. Error colors are diifferent. These are the ones that Paty handled me to put here

Reply:
We could probably adopt the same colours for the errors in that case, but don't worry about that for now.

How about we use `css={`...`}` for these instead?